components/RunStore.ts

```diff
@@ -82,13 +81,21 @@ export class RunStore {
 				if (isRepositoryDetailsComplete(repoDetails) && buildId && artifactName && filePath) {
 					const fixInVsCodeAction = {
 						text: 'Fix in VS Code',
-						linkUrl: `https://waveanalysis.microsoft.com/import?buildId=${buildId}&artifactName=${artifactName}&filePath=${filePath}&organization=${repoDetails.organizationName}&project=${repoDetails.projectName}&repoName=${repoDetails.repositoryName}&runIndex=${run._index}&resultIndex=${resultIndex++}&source=1esscans`,
+						linkUrl: `https://waveanalysis.microsoft.com/vscode/import?buildId=${buildId}&artifactName=${artifactName}&filePath=${filePath}&organization=${repoDetails.organizationName}&project=${repoDetails.projectName}&repoName=${repoDetails.repositoryName}&runIndex=${run._index}&resultIndex=${resultIndex++}&source=1esscans`,
 						imageName: 'vscode',
 						className: 'vscode-action'
 					}
 
+                    const fixInVsAction = {
+						text: 'Fix in Visual Studio',
+						linkUrl: `https://waveanalysis.microsoft.com/vs/import?buildId=${buildId}&artifactName=${artifactName}&filePath=${filePath}&organization=${repoDetails.organizationName}&project=${repoDetails.projectName}&repoName=${repoDetails.repositoryName}&runIndex=${run._index}&resultIndex=${resultIndex++}&source=1esscans`,
+						imageName: 'vs',
+						className: 'vs-action'
+					}
+
 					result.actions = [
-						fixInVsCodeAction
+						fixInVsCodeAction,
+                        fixInVsAction
```

Comment:
This indent looks off

Reply:
Think this might just be a GitHub thing with how they preview changes (per Monil) -- updated 4 spaces to a tab and it still looks wonky:

![image](https://github.com/user-attachments/assets/ba70b5d7-66df-4ab9-ac8b-f7ccbd5350de)

Gonna resolve this comment + merge bc outside of whitespace consistency (which now there is) not really sure if there's a fix, can loop back to this later if we feel like it's worth it